susemanager-utils/susemanager-sls/src/modules/uyuni_config.py

```diff
@@ -0,0 +1,1247 @@
+    :return: list of subscribed channels for the user
+    """
+    return UyuniChannel(uid, password).list_my_channels()
+
+
+def channel_software_set_user_manageable(channel_label, uid, access,
+                                         admin_user=None, admin_password=None):
+    """
+    Set the manageable flag for a given channel and user.
+    If access is set to 'true', this method will give the user manage permissions to the channel.
+    Otherwise, that privilege is revoked.
+
+    :param channel_label: label of the channel
+    :param uid: user login id
+    :param access: Flag which if user should have access to channel or not
+    :param admin_user: organization admin username
+    :param admin_password: organization admin password
+    :return: boolean indication success in operation
+    """
+    return UyuniChannelSoftware(admin_user, admin_password).set_user_manageable(channel_label, uid, access)
+
+
+def channel_software_set_user_subscribable(channel_label, uid, access,
+                                           admin_user=None, admin_password=None):
+    """
+    Set the subscribable flag for a given channel and user.
+    If value is set to 'true', this method will give the user subscribe permissions to the channel.
+    Otherwise, that privilege is revoked.
+
+    :param channel_label: label of the channel
+    :param uid: user login id
+    :param access: Flag which if user should subscribe a channel or not
+    :param admin_user: organization admin username
+    :param admin_password: organization admin password
+    :return: boolean indication success in operation
+    """
+    return UyuniChannelSoftware(admin_user, admin_password).set_user_subscribable(channel_label, uid, access)
+
+
+def channel_software_is_user_manageable(channel_label, uid, admin_user=None, admin_password=None):
+    """
+    Returns whether the channel may be managed by the given user.
+
+    :param channel_label: label of the channel
+    :param uid: user login id
+    :param admin_user: organization admin username
+    :param admin_password: organization admin password
+    :return: boolean which indicates if user can manage channel or not
+    """
+    return UyuniChannelSoftware(admin_user, admin_password).is_user_manageable(channel_label, uid)
+
+
+def channel_software_is_user_subscribable(channel_label, uid, org_admin_user=None, org_admin_password=None):
+    """
+    Returns whether the channel may be managed by the given user.
+
+    :param channel_label: label of the channel
+    :param uid: user login id
+    :param org_admin_user: organization admin username
+    :param org_admin_password: organization admin password
+    :return: boolean which indicates if user subscribe the channel or not
+    """
+    return UyuniChannelSoftware(org_admin_user, org_admin_password).is_user_subscribable(channel_label, uid)
+
+
+def channel_software_is_global_subscribable(channel_label, org_admin_user=None, org_admin_password=None):
```

Comment:
I'm following the convention `XML-RPC API namespace`+`_`+`XML-RPC Method`. So Since the method name is `isGloballySubscribable`, your suggestion for the name should be implemented.